### DISCUSSION TITLE
MaterialIconDarkTheme

### DISCUSSION THREAD

**Discussion by Dezexus** [anilibria/anilibria-winmaclinux#136] [Поделиться темой]
[AniLibriaQtDarkTheme.zip](https://github.com/anilibria/anilibria-winmaclinux/files/11423925/AniLibriaQtDarkTheme.zip)

**Comment by trueromanus** [comment]
Спасибо за тему, но нужно придумать уникальное имя для темы (MyDarkTheme точно не пойдет). Также укажите как бы Вы хотели значится в авторах (т.е. желаемый никнейм автора).

**Reply by Dezexus** [reply]
[MCMaterialIconDarkTheme.zip](https://github.com/anilibria/anilibria-winmaclinux/files/11429526/MCMaterialIconDarkTheme.zip)

Автор: Dezexus
